build-packages.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+usage()
+{
+    echo "Builds the NuGet packages from the binaries that were built in the Build product binaries step."
+    echo "No option parameters."
+    exit 1
+}
+
+working_tree_root="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+build_packages_log=$working_tree_root/build-packages.log
+arguments="$@"
+
+echo -e "Running build-packages.sh $arguments" > $build_packages_log
+
+# Parse arguments
+if [ $arguments == "-h" ] || [ $arguments == "--help" ]; then
+    usage
+fi
+
+echo -e "\n$working_tree_root/Tools/corerun $working_tree_root/Tools/MSBuild.exe $working_tree_root/src/packages.builds $arguments /nologo /v:minimal /flp:v=detailed;Append;LogFile=$build_packages_log" >> $build_packages_log
```

Comment:
I know this isn't unique to this script but I wanted to at least call attention to it. I'm not sure I like hard-coding the path to these tools directory in all our scripts we should think about options for sharing that information across scripts.